ntc_templates/templates/alcatel_sros_show_service_id_0_sap.textfsm

```diff
@@ -0,0 +1,29 @@
+Value PORT_ID (\S+)
+Value SERVICE_ID (\d+)
+Value INGRESS_QOS (\d+)
+Value INGRESS_FILTER (\S+)
+Value EGRESS_QOS (\d+)
+Value EGRESS_FILTER (\S+)
+Value ADMIN_STATUS (\S+)
+Value OPERATIONAL_STATUS (\S+)
+
+Start
+  ^=+\s*$$
+  ^SAP\(Summary\), Service \d+\s*$$ -> Header
+  ^. -> Error
+
+Header
+  ^=+\s*$$
+  ^PortId\s+SvcId\s+Ing\.\s+Ing\.\s+Egr\.\s+Egr\.\s+Adm\s+Opr\s*$$
+  ^\s+QoS\s+Fltr\s+QoS\s+Fltr\s*$$
+  ^-+\s*$$ -> Sap
+  ^. -> Error
+
+
+Sap
+  ^${PORT_ID}\s+${SERVICE_ID}\s+${INGRESS_QOS}\s+${INGRESS_FILTER}\s+${EGRESS_QOS}\s+${EGRESS_FILTER}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s*$$ -> Record
+  ^-+\s*$$
+  ^Number of SAPs : \d+\s*$$
+  ^-+\s*$$
+  ^=+\s*$$ -> End
+  ^. -> Error
```

Comment:
No real benefit from State transitioning.
Simple, less duplication, and more DRY to avoid all the transitions.

Please @bennnnnnnn, if you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
Start
  ^=+\s*$$
  ^SAP\(Summary\), Service \d+\s*$$
  ^PortId\s+SvcId\s+Ing\.\s+Ing\.\s+Egr\.\s+Egr\.\s+Adm\s+Opr\s*$$
  ^\s+QoS\s+Fltr\s+QoS\s+Fltr\s*$$
  ^${PORT_ID}\s+${SERVICE_ID}\s+${INGRESS_QOS}\s+${INGRESS_FILTER}\s+${EGRESS_QOS}\s+${EGRESS_FILTER}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s*$$ -> Record
  ^-+\s*$$
  ^Number\s+of\s+SAPs\s+:\s+\d+\s*$$
  ^. -> Error
```

Reply:
@bennnnnnnn
A bunch of good work here!
Thank you for your contributions to the community. :slightly_smiling_face: